### DISCUSSION TITLE
Maplibre Map works well under Google Colab but doesn't show under JupyterLab

### DISCUSSION THREAD

**Discussion by zffmzffm** [giswqs/geog-312#142] [Q&A] [answered]
Hello,

Maplibre Map works well under Google Colab but doesn't show under JupyterLab.
The variable "m" can be printed however the map won't show.

![image](https://github.com/user-attachments/assets/a046be00-b642-4049-a732-feeda60bb2a8)

The console shows the following errors:

![image](https://github.com/user-attachments/assets/c57de162-44c0-443a-a46d-0d5a10672934)

This problem is only for Maplibre maps under Jupyterlab. For ipyleaflet maps, there is no problem.

**Comment by giswqs** [comment, marked as answer]
This is a known issue. See https://github.com/developmentseed/lonboard/issues/640. 

Try creating a fresh conda env to install leafmap and maplibre.

```bash
conda create -n test python=3.12
conda activate test
conda install -c conda-forge leafmap maplibre jupyterlab
````
